mitodl/mit-open
# Do not analyze webpack by default

### What are the relevant tickets?
None

### Description (What does it do?)
This PR removes `WEBPACK_ANALYZE=True` from the `.env.example` file.

### How can this be tested?
Tests should pass.

If you want to see the effect:
1. `docker compose up` with `WEBPACK_ANALYZE=True`. Make a frontend change and time it with a stopwatch or something.
    - The `watch` container logs include the webpack output, which will report the time for rebuilds also. However, that doesn't seem to be the whole time it takes for browser to reload. For me, browser reload is a few seconds slower.
2. Repeat with `WEBPACK_ANALYZE` unset.
